spec/data/asset_audits/data.csv

```diff
@@ -0,0 +1,3 @@
+barcode,message,created_by,created_at
+SQPD-1,Destroying location,User1,2021-01-01 12:00:00
+SQPD-2,Destroying labware,User2,2021-01-02 12:00:00
```

Comment:
Fixture file names or the directory used for grouping them together could show the intention. 

spec/data/asset_audits/add_missing_asset_audit_records/
good_file.csv
badly_formatted.csv
no_data_section.csv
insufficient_columns.csv

etc.